<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-M42G5V1" ignored="0" name="Test" tests="6" failures="1" timestamp="26 Feb 2023 15:50:07 GMT" time="0.275" errors="0">
  <testcase name="test1" time="0.048" classname="failedTest.FailedTestExample"/>
  <testcase name="test2" time="0.016" classname="failedTest.FailedTestExample">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.009" classname="failedTest.FailedTestExample">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.031" classname="failedTest.FailedTestExample">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.004" classname="failedTest.FailedTestExample">
    <skipped/>
  </testcase> <!-- test2 -->
  <testcase name="test2" time="0.002" classname="failedTest.FailedTestExample">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at failedTest.FailedTestExample.test2(FailedTestExample.java:20)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Test -->
